docs/UserGuide.md

```diff
@@ -394,9 +420,33 @@ data file at the next run. Hence, it is recommended to take a backup of the file
 
 ## FAQ
 
-Q: How do I transfer my data to another Computer?
-A: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the
-data of your previous A&E home folder.
+**Q**: I double-clicked the Jar file and it doesnt run, why?\
+**A**: Make sure that Java 11 is installed and do refer to step 5 of [Quick-start](#quick-start) for instructions to run
+application.
+
+**Q**: I have a list of patients, how do i view their details?\
+**A**: Please click on the patient you are interested in viewing. The details will be on the right hand side in the
```

Comment:
I think you can use similar phrasing as lines 231-232 as the point is the same for consistency. Though this is just a minor suggestion and not really an issue